apps/coscrad-frontend/src/store/slices/resources/shared/notes-for-resource/self-note.presenter.tsx

```diff
@@ -21,12 +22,12 @@ export const SelfNotePresenter = ({ text, id, context }: SelfConnectionNote): JS
             </Grid>
             <Grid item>
                 {/* TODO Use property presenter helper after rebasing */}
-                <Typography variant="h5">Note ({id}):</Typography>
+                <Typography variant="h4">Note (Id: {id}):</Typography>
                 <Typography paragraph>{text}</Typography>
                 {/* TODO Eventually we should offer a way to highlight this context in the detail view */}
                 {/* We may not need to present this here in this way. This is for development only. */}
-                <Typography variant="h6">Context</Typography>
-                <Typography paragraph>{JSON.stringify(context)}</Typography>
+                <Typography variant="h5">Context:</Typography>
+                <EdgeConnectionContextPresenter context={context} />
```

Comment:
Add a todo to remove this later in favour of it being part of the detail presenter's API as mentioned above.

Reply:
done